lib/service/servicecfg/config.go

```diff
@@ -228,6 +230,9 @@ type Config struct {
 	// LoggerLevel defines the Logger log level.
 	LoggerLevel *slog.LevelVar
 
+	// LogConfig is the log configuration for handling logs from child processes.
+	LogConfig LogConfig
```

Comment:
Should we combine all log related fields? It feels a bit disjoint having the Logger and the LoggerLevel be separate.